cmd/server/http/http.go

```diff
@@ -561,3 +563,27 @@ func getGrafanaVarForEnv(env, dev, staging, prod string) string {
 	}
 	return ""
 }
+
+func notifyPolicyFailure(email, title, errorMessage, token string, userMappings map[string]string) {
+	client, err := slack.NewClient(token)
+	if err != nil {
+		log.Errorf("error initializing Slack Client in notifyPolicyFailure: %v", err)
```

Comment:
You need to return after this line to avoid continuing.
```suggestion
		log.Errorf("error initializing Slack Client in notifyPolicyFailure: %v", err)
		return
```